### ISSUE TITLE
missing legislation updates after Apr 26th

### ISSUE THREAD

**Issue by jehiah** [nysenate/NYSenate.gov-Website-D9#178]
I happened to notice that the URLs for legislation introduced on Apr 26th were not available even though they appeared in the Open Legislation API.

For example, the following URL 404's https://www.nysenate.gov/legislation/bills/2023/A9921

<img width="921" alt="image" src="https://github.com/nysenate/NYSenate.gov-Website-D9/assets/45028/b86ae221-f062-413b-87ed-929343f97d2f">

This affects assembly bills from A9882... onward and senate bills from S9104... onwards (and all updates on existing bills since the 26th).

https://legislation.nysenate.gov/bills/2023/A9882?amendment=&tab=Updates

<img width="1058" alt="image" src="https://github.com/nysenate/NYSenate.gov-Website-D9/assets/45028/06cc8f99-a868-4f84-ba3f-a05b4f2ae1d3">

I haven't seen this happen before so I wanted to flag it in case it was previously unknown.

**Comment by routinet** [comment]
Thank you for your report.  

I have confirmed bills 2023-A9221 and 2023-A9882 were not available on the web site despite being available in OpenLeg.  I have forced an import of those two bills, and they are now displaying as expected.  If you have discovered any other bills missing from our site, feel free to mention them here.

I am still investigating the underlying cause, and will update this issue when we have more information.

**Comment by routinet** [comment]
Hi @jehiah ,

We discovered a break in our communications with OpenLeg due to an update of our SSH infrastructure.  The underlying issue has been resolved, and our imports are running normally again.  We re-imported all updates suspected of not having been previously processed - a little over 414 bills were included.

Thanks again for reporting this.  I'm closing this issue as resolved.  If you encounter any other weirdness, feel free to start a new report.
